src/BlackSlope.Api.Common/Extensions/BlackSlopeServiceCollectionExtensions.cs

```diff
@@ -43,14 +43,13 @@ public static class BlackSlopeServiceCollectionExtensions
             services.AddMvc()
             .AddJsonOptions(options =>
             {
-                options.JsonSerializerOptions.IgnoreNullValues = true;
+                options.JsonSerializerOptions.DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull;
                 options.JsonSerializerOptions.PropertyNameCaseInsensitive = true;
                 options.JsonSerializerOptions.Converters
                     .Add(new JsonStringEnumConverter());
                 options.JsonSerializerOptions.Converters
                     .Add(new VersionJsonConverter());
-            })
-            .SetCompatibilityVersion(CompatibilityVersion.Version_3_0);

```

Comment:
5.x offered compatibility with 3.x. 6.x doesn't offer this, though I'd assume 7.x would.
Scrubbed.